src/app/Home/components/HomeComponent.js

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
 
-const HomeComponent = () => (
-  <h1 className='home__title'>Welcome to Andela Societies </h1>
-);
+import { connect } from 'react-redux';
 
-export default HomeComponent;
+export class App extends Component {
+  onIncrement = () => {
+    this.props.dispatch({
+      type: 'INCREMENT',
+    });
+  }
+
+  onDecrement = () => {
+    this.props.dispatch({
+      type: 'DECREMENT',
+    });
+  }
+
+  onIncrementAsync = () => {
+    this.props.dispatch({
+      type: 'INCREMENT_ASYNC',
+    });
+  }
+
+  render() {
+    return (
+      <div>
+        <h1 className='home__title'>Welcome to Andela Societies</h1>
+        <div className='home__container'>
+          <button type='button' onClick={this.onIncrement}>
```

Comment:
create a testSaga component or * and move this code there and just import the component in app index,

Reply:
I have created a stateless testSaga component for this.